app/models/subject.rb

```diff
@@ -3,5 +3,6 @@ class Subject < ApplicationRecord
 	has_many :teaching_classes, through: :sub_and_classes
 
 	validates :name, presence: true
+	validates :name, uniqueness: { case_sensitive: false, message: "Subject already exists." }
```

Comment:
@josephelias-rubyians   english , English are same. So it has to be be case sensitive